README.md

```diff
@@ -56,29 +56,28 @@
 
 ### Key Features <a name="key-features"></a>
 
-- **Fetch and render Missions & Rockets data.**
-- **Join/Leave Missions.**
-- **Reserve/Cancel reservation of Rockets.**
+- **Search for any city to check the weather conditions**
+- **Fetch and render the weather conditions**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Soon]()
+- [Watch live](https://check-currect-weather.onrender.com/)
```

Comment:
- [ ] For the live demo section, We need to make sure the app is deployed on Render, Netlify, or GitHub Pages. Also, we need to make sure all the functionalities are working well on the live version. Currently, I cannot open your live version and see the deployed app. Please make sure everyone can open and use your app live on the internet.

Reply:
- Well resolved 👍🏽